src/bindings/python/src/pyopenvino/core/remote_tensor.cpp

```diff
@@ -14,6 +14,13 @@ namespace py = pybind11;
 void regclass_RemoteTensor(py::module m) {
     py::class_<RemoteTensorWrapper, std::shared_ptr<RemoteTensorWrapper>> cls(m, "RemoteTensor");
 
+    cls.def(py::init([](RemoteTensorWrapper& tensor_wrapper, ov::Coordinate& begin, ov::Coordinate& end) {
```

Comment:
Could you please also cover all added functionality by tests [here](https://github.com/openvinotoolkit/openvino/blob/master/src/bindings/python/tests/test_runtime/test_remote_api.py)

Reply:
Added for copy_to/from methods for both RemoteTensor and RoiRemoteTensor